src/resolve/resolved-resolv-conf.c

```diff
@@ -328,6 +329,7 @@ int manager_write_resolv_conf(Manager *m) {
         _cleanup_ordered_set_free_ OrderedSet *dns = NULL, *domains = NULL;
         _cleanup_(unlink_and_freep) char *temp_path_uplink = NULL, *temp_path_stub = NULL;
         _cleanup_fclose_ FILE *f_uplink = NULL, *f_stub = NULL;
+        _cleanup_free_ char *fname = NULL;
```

Comment:
one more quest, can you move this to the innermost block this is used in? we generally try to minimize scope